src/main/java/nextstep/subway/path/application/PathService.java

```diff
@@ -25,14 +26,15 @@ public PathService(LineRepository lineRepository, StationService stationService,
         this.pathFinder = pathFinder;
     }
 
-    public PathResponse findShortestPath(long source, long target) {
+    public PathResponse findShortestPath(long source, long target, LoginMember loginMember) {
 
         Station sourceStation = stationService.findById(source);
         Station targetStation = stationService.findById(target);
 
         List<Line> lines = lineRepository.findAll();
 
         ShortestPath shortestPath = pathFinder.findShortestPath(lines, sourceStation, targetStation);
-        return new PathResponse(shortestPath);
+        int discount = AgeDiscount.discount(loginMember.getAge(), shortestPath.findFare());
+        return new PathResponse(shortestPath.findPaths(), shortestPath.findWeight(), discount);
```

Comment:
weight는 경로 게산을 하기 위한 알고리즘에 사용되는 용어이기 때문에 distance가 더욱 적합할 수 있겠네요 ! 🤔 
```suggestion
        return new PathResponse(shortestPath.findPaths(), shortestPath.distance(), discount);
```

Reply:
네넵 말씀하신대로 거리를 나타내는 distance가 메서드의 시그니처에 적절한 것 같습니다.